src/configure.ts

```diff
@@ -8,6 +13,14 @@ export type DescribeFunction = () => Promise<void> | void
  * Options provided to `configure` for setting up the test framework.
  */
 export interface ConfigureOptions {
+    /**
+     * A function called before all tests
```

Comment:
Please add a dot at the end of the sentence. This doc is shown to IDE users when autocompleting.

Reply:
Better: `A method called once per suite before all tests.`